tests/e2e/utils/utils.ts

```diff
@@ -0,0 +1,13 @@
+export function createPackageJson(filePath: string) {
```

Comment:
you can define here type of what this function returns `createPackageJson(filePath: string): Promise<null> { ... }`.